tests/Cache/ShardingTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Utopia\Tests;
+
+use Redis as Redis;
+use Utopia\Cache\Cache;
+use Utopia\Cache\Adapter\Redis as RedisAdapter;
+use Utopia\Cache\Adapter\Sharding;
+use Utopia\Tests\Base;
+
+class ShardingTest extends Base
+{
+    public static function setUpBeforeClass(): void
```

Comment:
Can we test here against the exception in the constructor regarding an empty array?